deploy/smcp-templates/base

```diff
@@ -4,26 +4,68 @@ spec:
   istio:
     global:
       tag: 1.0.2
+      # constrain resources for use in smaller environments
+      defaultResources:
+        requests:
+          cpu: 50m
+          memory: 128Mi
+        limits:
+          cpu: 50m
+          memory: 256Mi
       proxy:
         autoInject: disabled
         dnsRefreshRate: 300s
+        # constrain resources for use in smaller environments
+        resources:
+          requests:
+            cpu: 50m
+            memory: 128Mi
+          limits:
+            cpu: 50m
+            memory: 256Mi
       oauthproxy:
         tag: 4.1
         imagePullPolicy: IfNotPresent
       defaultPodDisruptionBudget:
         enabled: false
+    security:
+      # constrain resources for use in smaller environments
+      resources:
+        requests:
+          cpu: 100m
+          memory: 128Mi
+        limits:
+          cpu: 100m
+          memory: 256Mi
+    galley:
+      # constrain resources for use in smaller environments
+      resources:
+        requests:
+          cpu: 100m
+          memory: 128Mi
+        limits:
+          cpu: 100m
+          memory: 256Mi
     pilot:
+      resources:
+        requests:
+          cpu: 100m
```

Comment:
Tested CPU from the Jira was 150m

Reply:
pilot pod is composed of two containers
- istio-proxy (50m, 50m, 128Mi, 256Mi)
- discovery (100m, 100m, 128Mi, 256Mi)

Combined these give pod resources of 150m, 150m, 256Mi, 512Mi